### PR TITLE
fix(argus): Bump version to 0.5.2, bump scylla-driver to 3.28.8

### PR DESCRIPTION
**Pull request by k0machi**
This version fixes argus using inconsistent loggers to make it easier to separate its logs from sct events.

[Trello](https://trello.com/c/bEQpmUb7/4482-remove-specific-logger-for-testrun-and-use-name-instead)

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [x] ~New configuration option are added and documented (in `sdcm/sct_config.py`)~
- [x] ~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~
- [x]  All new and existing unit tests passed (CI)
- [x] ~I have updated the Readme/doc folder accordingly (if needed)~

### CODE REVIEW

**Comment by dkropachev** [comment]
Same PR - https://github.com/scylladb/scylla-cluster-tests/pull/4218

File: docker/env/version [hunk 1]
```diff
@@ -1 +1 @@
-1.01
+1.02-argus-logger-update
```
Comment (dkropachev) [inline docker/env/version:1]:
```suggestion
1.04
```

Comment (dkropachev) [reply docker/env/version:1]: 1.03 is already exists

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: docker/env/version (hunk 1 above)
Comment (k0machi) [reply docker/env/version:1]: Will do

Comment (k0machi) [reply docker/env/version]: Done

**Pull request merged**